package/kedro_viz/launchers/cli.py

```diff
@@ -87,10 +90,51 @@ def commands():
     help="Kedro configuration environment. If not specified, "
     "catalog config in `local` will be used",
 )
-def viz(host, port, browser, load_file, save_file, pipeline, env):
+@click.option(
+    "--autoreload",
+    is_flag=True,
+    help="Autoreload viz server when a Python file change in the Kedro project",
+)
+def viz(host, port, browser, load_file, save_file, pipeline, env, autoreload):
     """Visualise a Kedro pipeline using Kedro viz."""
     try:
-        run_server(host, port, browser, load_file, save_file, pipeline, env)
+        if autoreload:
+            is_localhost = host in ("127.0.0.1", "localhost", "0.0.0.0")
```

Comment:
I notice that this `is_localhost` is duplicated in a couple of places in `viz` -- any chance to standardise it and define it once and pass it along, or at least move it into a utility rather than redefining the tuple of localhost-equivalents? `is_localhost(host)`?